topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorMockTest.java

```diff
@@ -283,21 +274,16 @@ private Config waitUntilTopicInKafka(String topicName, Predicate<Config> p) {
     }
 
     void reconcile(VertxTestContext context) throws InterruptedException {
-        Checkpoint async = context.checkpoint();
         session.topicOperator.reconcileAllTopics("test").onComplete(ar -> {
             if (!ar.succeeded()) {
                 context.failNow(ar.cause());
             }
-            async.flag();
         });
-        if (!context.awaitCompletion(60, TimeUnit.SECONDS)) {
```

Comment:
This suggests we expected reconcile to block until the reconciliation was finished, but the new code doesn't do that. Given that `awaitCompletion()` was probably not doing what we expected maybe it's not a problem, but maybe some assumptions in the test about the blocking behaviour are now being violated (or always were).

Reply:
Good point. I'll add some blocking behaviour.